spec/spec_helper.rb

```diff
@@ -30,7 +28,7 @@
   config.include SqlTestHelpers
 
   config.after do
-    data_sources.each do |table|
+    ActiveRecord::Base.connection.tables.each do |table|
```

Comment:
Should this be `ActiveRecord::Base.connection.data_sources`?

Reply:
@jchuang Oops, thanks. It looks like it might not matter as much on these versions of ActiveRecord and for this test suite, but I'll switch it back to `data_sources`.